src/main/java/mate/academy/spring/validation/Email.java

```diff
@@ -0,0 +1,18 @@
+package mate.academy.spring.validation;

```

Comment:
<img width="289" alt="image" src="https://github.com/mate-academy/jv-spring-security/assets/72043323/c5cf7bc2-aa32-42b0-8923-78f340c5ce7e">

Did you forget to commit this file? (EmailValidator)